### DISCUSSION TITLE
How to get medias by using specific tag?

### DISCUSSION THREAD

**Discussion by karorox** [sbertix/Swiftagram#202] [Q&A] [answered]
Currently I'm trying to use Swiftagram 5.0.3

I've read README, Example code and past issues, but finally I couldn't get it how to fetch medias by using specific tag like "travel". 

Could you give me example code?

**Comment by sbertix** [comment, marked as answer]
Hey @karorox, 

I've moved this to the _Discussions_ tab as this is not an issue, please make sure you're starting support requests from there next time 😊 

You can fetch tag-specific posts like so:

```swift
// Assuming you already have the `Secret`.
let secret: Secret = /* some `Secret` */
// Assuming this is the name of the tag.
let tag: String = "travel"
// Assuming this is retained.
var bin: Set<AnyCancellable> = []

Endpoint.tag(tag)
   .posts
   .recent   // or use `.top` for top posts.
   .unlock(with: secret)
   .session(.instagram)   // or pass your custom `URLSession`.
   .pages(1)   // to only fetch the first page.   
   .sink(receiveCompletion: { _ in }, receiveValue: { print($0) })
   .store(in: &bin)
```
